### DISCUSSION TITLE
Default email for initial login

### DISCUSSION THREAD

**Discussion by thenugraha** [lablup/backend.ai#1574] [Q&A]
What is the default email and password to login when I have installed it with ``` install-dev.sh```? And how can I access the files which have that information? Thank you!

**Comment by achimnol** [comment]
They are populated from initial fixtures [here](https://github.com/lablup/backend.ai/tree/main/fixtures/manager).

For example, the local admin account is: `admin@lablup.com` / `wJalrXUt`.
